src/utils/_extends.scss

```diff
@@ -0,0 +1,5 @@
+%cell-text-settings {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  display: inline-block;
+}
```

Comment:
It's good that you know how to use it. But do you really need this extension here? You use it only once, you could write this style inside the body. 
You don't need to fix it now, just a comment for the next tasks so you don't do extra work when it's not needed